scripts/gulp/tasks/vr-build.ts

```diff
@@ -0,0 +1,21 @@
+import { task, series } from 'gulp';
```

Comment:
Split up `screener.ts` into `vr-test.ts` and `vr-build.ts` to better reflect the commands since they are actually used as `vr:build`  and `vr:test`.

The two steps have nothing in common with each other either